test/test_to_code_mapping_test.rb

```diff
@@ -5,7 +5,7 @@ class TestToCodeMappingTest < TTNT::TestCase
   def setup
     # clean up generated .ttnt files
     FileUtils.rm_rf("#{@repo.workdir}/.ttnt")
```

Comment:
Fixed in this too: https://github.com/Genki-S/ttnt/commit/1ec9320e6d6dff2aaa6e1d42fe42807477255e2c